src/components/IButtonGroup/IButtonGroup.vue

```diff
@@ -52,24 +43,26 @@ const props = defineProps({
      */
     size: {
         type: String,
-        default: ''
+        default: undefined
     }
 });
 
-const size = useSize({ componentName, currentSize: props.size || buttonGroup.size });
+const componentSize = useComponentSize({ componentName, currentSize: props.size });
```

Comment:
Don't forget to take parent `buttonGroup.size` into account here for nested groups.